_data/products.yml

```diff
@@ -1,8 +1,8 @@
-serverless: CockroachCloud Free (beta)
+serverless: CockroachCloud Serverless (beta)
 # to be CockroachCloud Serverless then CockroachDB Serverless (beta)
-serverless-plan: CockroachCloud Free #i.e. "1. Choose the **CockroachCloud Free** plan."
+serverless-plan: CockroachCloud Serverless #i.e. "1. Choose the **CockroachCloud Free** plan."
```

Comment:
Why do we need a separate variable here? Won't the `serverless` variable work fine?

Reply:
There are a decent amount of places where it feels redundant/clunky to keep saying "(beta)" such as tabs or in longer blocks of text. Hopefully this can be removed once we're out of beta.